deployment/terraform/utils.go

```diff
@@ -352,20 +355,114 @@ func getServerURL(output *Output, deploymentConfig *deployment.Config) string {
 	return deploymentConfig.ServerScheme + "://" + url
 }
 
-// GetAWSConfig returns the AWS config, using the profile configured in the
-// deployer if present, and defaulting to the default credential chain otherwise
-func (t *Terraform) GetAWSConfig() (aws.Config, error) {
+// Durations for the expiry of the AWS Role credentials and for its refresh
+// interval, which, out of an abundance of caution, is a bit shorter than half
+// the expiry duration of the role.
+const (
+	// One hour is the maximum allowed by role chaining
+	AWSRoleTokenDuration        = 1 * time.Hour
+	AWSRoleTokenRefreshInterval = 25 * time.Minute
```

Comment:
Can we do `AWSRoleTokenRefreshInterval = AWSRoleTokenDuration/2 ` for simplicity? Also, are we really gaining anything by reducing the interval by a further 5 minutes? Or is it just a peace of mind thing? If we really want to reduce further than half, I'd just prefer to run it 4 times per hour. Every 25 minutes looks a bit odd to me.